src/pages/Home/Home.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const HomePage = () => {
+  return (
+    <div className="page page-home">
+      <h1>Arvin T. Viar</h1>
+      <h2>React Training</h2>
+    </div>
+  );
+}
+
+export default HomePage;
```

Comment:
We can also apply our changes from the Player component here -- remove the curly body brackets and return but keep the parenthesis.